apps/site/src/views/Landing/components/ApplyButton/ApplyButton.module.scss

```diff
@@ -0,0 +1,33 @@
+@use "bootstrap-utils" as utils;
+
+.applyButtonContainer {
+	position: relative;
+}
+
+.applyButton {
+	position: relative;
+	transform: skew(-30deg);
+	height: 80px;
+	width: 250px;
+	margin: 0px;
+	border: none;
+	background-color: utils.$yellow;
+	transition: all 0.1s ease;
```

Comment:
Suggestion: transitioning all properties may lead to unexpected behavior. It may be better to specify which properties should undergo transitions.

Reply:
One note: I used `text-decoration: underline` to underline on hover, but a transition cannot be applied to it.